tests/_test_core_strings.py

```diff
@@ -188,3 +193,168 @@
 
 }); // END REQUIRE.JS block
 </script>"""
+
+STANDALONE_HTML_STRING = """
```

Comment:
I agree we can do either or, I decided to do strings because that's what
we already had.

On (Jun-28-16|10:42), Antonio Gonzalez wrote:
> > @@ -188,3 +193,168 @@
> >  
> >  }); // END REQUIRE.JS block
> >  </script>"""
> > +
> > +STANDALONE_HTML_STRING = """
> 
> Should these be separate files or are we OK having them here as text? I can see pros/cons for both options ... 
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/510/files/378f456b29428865360dc7c3ad7a0beaed9e08b2#r68805877